website/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "scripts": {
+    "start": "docusaurus-start",
+    "build": "docusaurus-build",
+    "publish-gh-pages": "docusaurus-publish",
+    "write-translations": "docusaurus-write-translations",
+    "version": "docusaurus-version",
+    "rename-version": "docusaurus-rename-version"
+  },
+  "devDependencies": {
+    "docusaurus": "^1.4.0"
```

Comment:
@sveinpg 

Change this line to:

```
"docusaurus": "git://github.com/facebook/docusaurus.git#127500532103ee9096de9f202866546969ea107d"
```

Refer to https://stackoverflow.com/a/18049685/1751946 for more info.